src/KM_Controls.pas

```diff
@@ -5339,13 +5369,16 @@ procedure TKMColumnBox.DoPaintLine(aIndex: Integer; X, Y: Integer; PaintWidth: I
       end else
       begin
         TextSize := gRes.Fonts[fFont].GetTextSize(Rows[aIndex].Cells[I].Caption);
-        if HighlightOnMouseOver and (csOver in State) and (fMouseOverRow = aIndex) then
-          Color := Rows[aIndex].Cells[I].HighlightColor //Brighten(Rows[aIndex].Cells[I].Color)
-        else
-        if HighlightError and (aIndex = ItemIndex) then
+        if not aDoNotHighlight

```

Comment:
aHighlight is not correct here, because we may highlight, or may not, depends on other reasons.
Probably aAllowHighlight is Ok here